airflow/models/dagcode.py

```diff
@@ -147,7 +148,7 @@ def has_dag(cls, fileloc: str, session=None) -> bool:
         :param session: ORM Session
         """
         fileloc_hash = cls.dag_fileloc_hash(fileloc)
-        return session.query(exists().where(cls.fileloc_hash == fileloc_hash)).scalar()
+        return session.query(literal(True)).filter(cls.fileloc_hash == fileloc_hash).one_or_none() is not None
```

Comment:
is literal(True) == exists() here ? can you point me to docs please

Reply:
mssql doesn't support exists in select query , i.e it doesn't support

```
SELECT EXISTS (SELECT *
FROM dag_code
WHERE dag_code.fileloc_hash = 15056821532142474) AS anon_1
```

https://github.com/apache/airflow/pull/9973#issuecomment-665492863

Reply:
`return session.query(literal(True)).filter(cls.fileloc_hash == fileloc_hash).first() is not None`
instead of 
`return session.query(literal(True)).filter(cls.fileloc_hash == fileloc_hash).one_or_none() is not None` ?

I mean is it expected to have multiple duplicate fileloc_hash ?